ReactiveCocoa/NSObject+Association.swift

```diff
@@ -1,5 +1,29 @@
 import ReactiveSwift
 
+internal struct AssociationKey {
```

Comment:
Just being a nicer citizen to the memory allocator, instead of having six scattered bytes that fragments the address space.